Week 1/Extra Practice Problems (Week 1)/0503: Next Greater Element II/elijah.cpp

```diff
@@ -0,0 +1,25 @@
+class Solution {
+public:
+    vector<int> nextGreaterElements(vector<int>& nums)
```

Comment:
Nice :) there's also a faster algorithm btw!

Feel free to chat with @DominicS99 on a better approach here